ci/gpu/build.sh

```diff
@@ -59,6 +59,7 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=${MINOR_VERSION}" \
+      "pytest-timeout" \
```

Comment:
@dantegd, just a heads up. we did just recently merge a PR to add `pytest-timeout` to the gpuci-build-environment here https://github.com/rapidsai/gpuci-build-environment/pull/73, so you should be able to remove this package from your end without any issues. additionally, we added a `PYTEST_TIMEOUT=300` environment variable on the gpu build jobs in Jenkins, but I don't think there's any issue with you guys specifiying your own timeout in the flags below.

Reply:
(keep the conda recipe `pytest-timeout`packages though)

Reply:
thanks for the heads up! I’ll keep the timeout explicitly on the pytest call just for safety initially, but could remove it in the near future if we don’t see a need to change the time value for some time